pywbem/_server.py

```diff
@@ -762,6 +764,34 @@ def get_central_instances(self, profile_path, central_class=None,
             implementation supports only the scoping class methodology.
             `None` will cause the scoping class methodology not to be attempted.
 
+          reference_direction (:term:`string`):
+            Defines the usage of the ResultRole parameter in the association
```

Comment:
I think it is easier to understand if we describe the difference in terms of the navigation direction across the CIM_ReferencedProfile association, i.e. navigating to the component profile in direction of Antecedent (DMTF) or Dependent (SNIA). The ResultRole parameter of the Associators operation does not need to be mentioned at all that way.

Reply:
DONE

Reply:
Good call, use the phrase 'navigation direction'